feelpp/feel/feelmodels/modelexpression.hpp

```diff
@@ -112,15 +112,21 @@ public :
     }
     bool isConstant() const { return this->isEvaluable(); }
 
-    evaluate_type evaluate() const
+    template <typename TheSymbolExprType = symbols_expression_empty_t>
+    evaluate_type evaluate( TheSymbolExprType const& se = symbols_expression_empty_t{} ) const
```

Comment:
why not use universal reference ? I am not sure about default types/values though

Reply:
maybe, but generally, SymbolsExpr are not passed by rvalue